lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -186,10 +186,20 @@ def write_ssh_key
       def start_ansible_runner
         env = {}
         env['FOREMAN_CALLBACK_DISABLE'] = '1' if @rex_command
```

Comment:
This is now unused

Reply:
It is still used here https://github.com/theforeman/smart_proxy_ansible/pull/66/files#diff-65b556454b9717ae33c08487e6ace03409dadb3c66b54f921b1dee1caccfb142R202

Reply:
Ah, my bad.